quil-rs/src/instruction/extern_call.rs

```diff
@@ -295,6 +296,11 @@ impl ExternPragmaMap {
         )
     }
 
+    /// Extend the [`ExternPragmaMap`] with another.
```

Comment:
```suggestion
    /// Extend this [`ExternPragmaMap`] with another.
```
We should also document which map wins if there's a collision of keys.